### ISSUE TITLE
QBatchNormalization: Progress

### ISSUE THREAD

**Issue by asti205** [google/qkeras#3]
First: Thanks a lot for this wonderful package, it is really interesting!

Secondly, I wanted to ask you about the progress on implementing QBatchNormalization. In readme.md, you were stating that:

> Finally, QBatchNormalization is still in its experimental stage, as we have not seen the need to use this yet due to the normalization and regularization effects of stochastic activation functions.

Did you implement a "Beta" version of this experimental version? I know, most of the time, there are not many parameters in the BatchNormalization Layer, nonetheless it would be interesting to study the effects of quantizing BatchNormalization Layers of a pretrained model like MobileNetV2, which includes BatchNormalization Layers.

Thanks a lot,
asti205

**Comment by zhuangh** [comment]
We have been working on the QBatchNormalization and will push the enhancement soon after finishing the tests. (Note: the enhancement will use tensorflow 2.0 and tf.keras.) Stay tuned. Thanks!

--
Hao Zhuang
Software Engineer @ Google

**Comment by zhuangh** [comment]
Since this [commit](https://github.com/google/qkeras/commit/666da8f44b2329a054a882e51a9308573d25f4b9), we have added QBatchNormalization

**Comment by zhuangh** [comment]
close this issue since it has been quiet for a while. feel free to reopen it if anything else comes up.
